tests/cpu/test_rn50_cpu_ops.py

```diff
@@ -416,6 +416,9 @@ def test_view(self):
         self.assertRaises(RuntimeError, lambda: tensor.view(7, -1))
         self.assertRaises(RuntimeError, lambda: tensor.view(15, -1, -1))
 
+        # TODO(Eikan): DNNL OP does not support >6 dim tensor, so we disable it temporily. When we fix it, we will open it
+        old_dnnl_conf = ipex.get_auto_dnnl()
```

Comment:
May consider to do `with` with context manager to save the complexity of save/restore original conf.

Reply:
Great! I will fix it.

Reply:
It seems like that "with" does not work for native C++ API.